### ISSUE TITLE
The focus indicator does not display properly for some of the headings in the navigation menu

### ISSUE THREAD

**Issue by jakubmiarka** [alphagov/tech-docs-gem#114]
Under certain circumstances* the headings in the sidebar (which are collapsible) are not wrapped within `<span>` causing the heading not to be highlighted when focused/tabbing through. Failing the WCAG 2.1. success criterion 2.4.7 (focus visible).

See the example [here](https://www.docs.verify.service.gov.uk).

*I think it's the combination of the `toc_level` config and a use of nested pages

This fails [WCAG 2.1 success criterion 2.4.7 (focus visible)](https://www.w3.org/WAI/WCAG21/quickref/#focus-visible).

**Comment by nickcolley** [comment]
Can reproduce this on the tech docs template website too: https://tdt-documentation.london.cloudapps.digital/#technical-documentation-template
